### ISSUE TITLE
Does the score vary to the seed setting? 

### ISSUE THREAD

**Issue by deepffff** [real-hjq/MS-SWD#1]

**Comment by real-hjq** [comment]
Yes, the scores will change with the random weights, but the change will be very slight.
We have also uploaded a learned version of MS-SWD on IQA-PyTorch, and the scores will not change.
